Samples/LambdaLang/Program.cs

```diff
@@ -6,26 +6,6 @@
 using SemanticAlgebra.Option;
 using System.Collections.Immutable;
 
-
-// var s = Option.Some(40);
```

Comment:
When deleting those test commented code, ensure to there is corresponding unit tests exists. If not, add them.